counterpartylib/test/fixtures/vectors.py

```diff
@@ -1618,6 +1618,12 @@
             'comment': 'referencing parent asset by name composes a reissuance',
             'in': (ADDR[0], None, 'PARENT.already.issued', 1000, True, ''),
             'out': ('mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc', [], b'\x00\x00\x00\x14\x01S\x08!g\x1b\x10e\x00\x00\x00\x00\x00\x00\x03\xe8\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00')
+        }, {
+            'comment': 'basic child asset with compact message type id',
+            'mock_protocol_changes': {'short_tx_type_id': True},
+            'in': (ADDR[0], None, 'PARENT.child1', 100000000, True, ''),
+            'out': ('mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc', [], bytes.fromhex('1501530821671b10010000000005f5e100010a57c6f36de23a1f5f4c46'))
+            # 15|01530821671b1001|0000000005f5e100|01|0a|57c6f36de23a1f5f4c46
```

Comment:
I added one specific message test (an issuance) with the short_tx_type_id protocol change.